test/common/listener_manager/listener_impl_integration_test.cc

```diff
@@ -0,0 +1,34 @@
+#include "envoy/network/address.h"
+
+#include "source/server/config_validation/server.h"
+
+#include "test/integration/server.h"
+#include "test/mocks/server/options.h"
+#include "test/test_common/environment.h"
+#include "test/test_common/file_system_for_test.h"
+#include "test/test_common/thread_factory_for_test.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
+// Regression test for https://github.com/envoyproxy/envoy/issues/28413
+TEST(ConfigValidateTest, ValidateGood) {
```

Comment:
Can you move this to listener_impl_test.cc, please?

Reply:
Alternatively, rename it to `listener_impl_test.cc`, since it's not an integration test.